outsettingsvalues/templatetags/custom_filter.py

```diff
@@ -0,0 +1,28 @@
+"""
+File with custom tags.
+"""
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def checktype(args):
+    """
+    Check type of args. Dict or not.
+
+    :param args:
+    :return: True or False
```

Comment:
For my point of view, here must be a description when the function returns true and when false. Or in case, when this description is trivial - remove it please.